support-e2e/tests/cron/ad-lite.test.ts

```diff
@@ -19,14 +19,14 @@ test.describe('Ad Lite Checkout', () =>
 				`/uk/guardian-ad-lite`,
 				{
 					context,
-					baseURL,
+					baseURL: baseURL,
```

Comment:
Has the shorthand been removed here because eslint didn't like it? I'm surprised about that - are we definitely using the Guardian eslint config?